README.md

```diff
@@ -4,6 +4,61 @@ This package provides an way of generating files using Dart code, outside of
 `pub`. These files are generated directly on disk in the current package folder,
 and rebuilds are _incremental_.
 
+## Differences between the `build` package and `pub`.
+
+You might be asking, why use this package instead of a pub `Transformer`? There
+are a few key differences that make this package better for some use cases.
+
+### Outputs
+
+Pub will only ever output files into your `build` directory, and pub serve only
+ever outputs them into an in memory file system. With the `build` package, you
```

Comment:
I guess "in-memory" would make this more readable. I don't know if this would make it worse English. My English knowledge isn't too good, but I read 3 times until it became clear how "in" fits into that sentence.